CCPAConsentViewController/Classes/UserConsent.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 /// Indicates the consent status of a given user.
-@objc public enum ConsentStatus: Int, Codable {
+@objc public enum ConsentStatus: Int, RawRepresentable, Codable {
```

Comment:
Nice to learn about the `RawRepresentable` protocol. I think we don't need to explicitly conform to it though:
> For any enumeration with a string, integer, or floating-point raw type, the Swift compiler automatically adds RawRepresentable conformance. 
https://developer.apple.com/documentation/swift/rawrepresentable

Did you do that so we can continue to use `Int` as its "parent" but use `String` as its raw value?

Reply:
Swift enums can be exported to Objective-C using the @objc attribute. @objc enums must declare an integer raw type.
We can use String as raw type then we have to remove the @objc attribute as in Objective-C we don't have enum with "String" as parent. 
@andresilveirah  if you are fine with removing @objc attribute from enum declaration then I can use string as enum parent, if we want to keep swift enum exportation in objective-c then we can go with this approach.

Reply:
Got it @vilasma . What I'm pointing out is the fact that Swift will, by default, make any enums that inherit from a raw type conform to `RawRepresentable`. At least that's what I understood from their doc posted above.

Reply:
Got it @andresilveirah,  removed from enum declaration.